modules/serverless-event-delivery-admin-config.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * /serverless/event_workflows/serverless-event-delivery.adoc
+
+[id="serverless-event-delivery-admin-config_{context}"]
```

Comment:
This is gcp broker specific, none of the components in our product support this cluster global config.  Although the single fields docs make sense

Reply:
Cool, thanks for confirming, that actually simplifies things